### ISSUE TITLE
Gadget grabbable takes priority over gadget scanner

### ISSUE THREAD

**Issue by alandao** [aardvarkxr/aardvark#161]
It's possible to have a composed entity with a moveable component whose grabbable range is greater than the immediate surrounding. This means the hand-cursors will always be selected to grab entity rather than anything else. I can't delete the gadget instance using the gadget scanner, as trying to press the trash button ends up grabbing the entity (assuming you have one hand that picked up the entity, the other hand picking up the scanner, and then letting go of the entity to press the trash button). To mitigate this, I have to restart Aardvark.

I think the gadget scanner and its grabbable entities(the trash button) should have precedence over any other gadget out there, so it can always be possible to delete rogue apps.

**Comment by JoeLudwig** [comment]
Yeah, the system stuff should probably have higher priorities on its volumes and forbid gadgets from using priorities that high.
